Unix/server/server.c

```diff
@@ -166,6 +168,122 @@ static char** _DuplicateArgv(int argc, const char* argv[])
     return newArgv;
 }
 
+
+//
+// Duplicates and sets needed variables and removes the rest
+//
+static char** _DuplicateEnvp(const char* envp[])
+{
+    int i;
+
+    // Special env variables:
+    const char Krb5_Trace[]  = "KRB5_TRACE=/dev/stderr";
+    const char Krb5_KTName[] = "KRB5_KTNAME=";
+    const char Krb5_CCName[] = "KRB5_CCNAME=";
+    const char Ntlm_User_File[] = "NTLM_USER_FILE=";
+
+    int env_count = 0;
+    char *envitem = NULL;
+    for (envitem = (char*)(envp[0]); envitem; envitem = (char*)(envp[env_count]))
+    { 
+        env_count++;
+    }
+
+    char **newEnv = (char**)PAL_Malloc((env_count+3)*sizeof(char*));
```

Comment:
looks like this should be env_count+4 ?
Krb5_Trace
Krb5_KTName
Krb5_CCName
Ntlm_User_File